jakartaee/10/webprofile/23.0.0.3-Java11-TCKResults.adoc

```diff
@@ -0,0 +1,1764 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of Jakarta EE Platform 10.
+
+== Open Liberty 23.0.0.3 Jakarta EE Platform 10 using Java 11 Certification Summary
+
+* Product Name, Version and download URL (if applicable):
++
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/23.0.0.3/openliberty-webProfile10-23.0.0.3.zip[Open Liberty 23.0.0.3]
+
+* Specification Name, Version and download URL:
++
+https://jakarta.ee/specifications/webprofile/10[Jakarta EE Web Profile 10]
+
+* TCK Version, digital SHA-256 fingerprint and download URL:
++
+https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.2.zip[Jakarta EE Platform TCK 10.0.2],
```

Comment:
I believe this is correct as is, but please confirm

Reply:
This is indeed correct. For Web Profile, you run the Platform TCK with an extra parameter to restrict it to WP-appropriate tests.